models/Recipe.model.js

```diff
@@ -3,6 +3,39 @@ const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
   // TODO: write the schema
+  title: {
+    type: String,
+    unique: true,
+  },
```

Comment:
```suggestion
  title: {
    type: String,
    unique: true,
    required: true
  },
```
Hay que leeer el enunciaaado